02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,51 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Kevin Kerner</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <h1>Kevin Kerner</h1>
+    <h6 style="">1604 High Lonesome Leander, TX  78641</h6>
+
+  <h2>Bio</h2>
+  <p>Lorem style ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+  <h2>Hobbies</h2>
+      <ol>
+          <li>Chasing Kids Around</li>
+          <li>Eating Fries</li>
+          <li>Design and Development</li>
+      </ol>
+
+  <h2>Things I Don't Like</h2>
+      <ul>
+        <li>Liver</li>
+        <li>List element One</li>
+        <li>List element One</li>
+      </ul>
+
+  <h2>TV Shows I like to Watch</h2>
+  <figure>
+  <a href="http://www.amc.com/shows/the-walking-dead" target="_blank"><img src="https://pbs.twimg.com/profile_images/687320361201987584/4UdE2wWe_400x400.jpg"</></a>
```

Comment:
Yep... was going to clean it up over the weekend and add the other homework elements too.  Pretty fun stuff.